packages/api-tests/README.md

```diff
@@ -0,0 +1,8 @@
+# ContainerJS Testing
+This project contains the Spectron UI tests for the OpenFin and Electron APIs
+
+## Testing
+```
+npm i
+npm run test:ui
```

Comment:
Ah - fair point - might be worth adding a quick note to this readme with that information.